R/utils.R

```diff
@@ -13,6 +13,14 @@ check_data_columns =  function(data, id = 'id', time = 'time', gl = 'gl'){
                    " are not present in the data")
       stop(msg)
     }
+    if (time_check) {
+      if (lubridate::is.POSIXct(data$time)){ # Check if already in date format
```

Comment:
better to not have empty action
if !(lubridate::is.POSIXct(data$time){
 # action here
}